src/routes/jira/events/jira-events-install-post.ts

```diff
@@ -7,16 +7,17 @@ import { metricHttpRequest } from "config/metric-names";
  * Handle the install webhook from Jira
  */
 export const JiraEventsInstallPost = async (req: Request, res: Response): Promise<void> => {
-	req.log.info("Received installation payload");
 
 	const { baseUrl: host, clientKey, sharedSecret } = req.body;
+	req.log.info({ jiraHost: host, clientKey },  "Received installation payload");
```

Comment:
`clientKey` is public information, so didn't bother hash it. `jiraHost` is already hashed.